test/es6/rlexe.xml

```diff
@@ -1059,6 +1059,13 @@
       <tags>BugFix</tags>
     </default>
 </test>
+<test>
+    <default>
+      <files>ES6NewTarget_bugfixes.js</files>
+      <compile-flags>-ES6Classes -forceundodefer -args summary -endargs</compile-flags>
```

Comment:
forceundodefer works for top level global function - so the test you have implemented in es6newtarget_bugfixes.js is not exercising the scenario.